helm-charts/strimzi-kafka-operator/templates/020-RoleBinding-strimzi-cluster-operator.yaml

```diff
@@ -16,3 +17,25 @@ roleRef:
   kind: ClusterRole
   name: strimzi-cluster-operator-namespaced
   apiGroup: rbac.authorization.k8s.io
+{{ $root := . }}
+{{ range .Values.strimzi.watch_namespaces }}
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: RoleBinding
+metadata:
+  name: strimzi-cluster-operator
+  labels:
+    app: {{ template "strimzi.name" $root }}
+    chart: {{ template "strimzi.chart" $root }}
+    component: role-binding
+    release: {{ $root.Release.Name }}
+    heritage: {{ $root.Release.Service }}
+subjects:
+  - kind: ServiceAccount
+    name: strimzi-cluster-operator
+    namespace: {{ . }}
+roleRef:
+  kind: ClusterRole
+  name: strimzi-cluster-operator-namespaced
+  apiGroup: rbac.authorization.k8s.io
+{{ end }}
```

Comment:
I don't like the duplication happening here, but alternatives weren't nicer either:

we could create a general RoleBinding template in _helpers.tpl and use it here or we could add the `if not empty logic` to each of the RoleBinding templates 

Let me know what you prefer and I can update the PR

Reply:
I'm not sure I understand the concern.  Can't this whole resource be defined in a loop?

Reply:
I don't like having the RoleBinding defined twice in the template, but in order to iterate over `Release.namespace` + `strimzi.watch_ watch_namespaces` we will need to add control logic for the case where `strimzi.watch_namespaces` is empty or helm would templatize `[]`

Reply:
Yes, agreed, but doesn't the way `strimzi.watched_namespaces` is defined right now guarantee that a list with at least one namespace (`Release.namespace`) will always exist?  Then the whole resource body can be within the loop and will always iterate at least one time.  Maybe I'm misinterpreting that `_helper` definition..

Reply:
Yes, it would make more sense, I tried defining a variable in the `_helpers.tpl` I could use later in the RoleBindings `range` but couldn't get it working :( 

Not sure if it's a restriction in how helm `_helpers.tpl` works or that I'm just a helm noob :)